.github/renovate.json

```diff
@@ -106,6 +106,46 @@
       "matchManagers": ["npm", "nvm"],
       "additionalBranchPrefix": "fe-"
     },
+    {
+      "description": "Exclude frontend deps until Optimzie migrate to react-router v6 and react v18",
+      "matchManagers": ["npm", "nvm"],
+      "matchFileNames": ["optimize/**"],
+      "matchPackagePatterns": [
+        "@carbon/react",
+        "old-bpmn-js",
+        "react",
+        "react-dom",
+        "react-router",
+        "react-router-dom",
+        "@types/react",
+        "@types/react-dom",
+        "@types/react-router-dom"
+      ],
+      "enabled": false
+    },
+    {
+      "description": "Exclude Optimzie backend dependencies",
+      "matchManagers": ["maven"],
+      "matchFileNames": ["optimize/**"],
+      "matchPackagePatterns": [
+        "org.camunda.optimize:optimize-data-generator",
+        "org.camunda.optimize:camunda-optimize",
+        "org.camunda.optimize:util",
+        "org.camunda.optimize:qa",
+        "org.camunda.optimize:optimize-plugin-parent",
```

Comment:
plugins will be disabled soon, right @PHWaechtler?

Reply:
They're on the delete list ♻️